docs/releasenotes/archive/service/year2020.md

```diff
@@ -5,4 +5,662 @@ sidebar_label: 2020 Archive
+
+Update - Just wanted to let you know that we've made an underlying change to our [XML Parsing](https://help.sumologic.com/05Search/Search-Query-Language/01-Parse-Operators/07-Parse-XML-Formatted-Logs "Parse XML Formatted Logs") library for parsing XML-formatted logs. The following minor behavioral chances will change how results are returned from parsing XML log:
+
+1\. Empty nodes will be returned as self closing tags
+
+For example - Consider the following xml log line (see last empty tag)\
+`<users><user id="emptytag" role="manager"><first_name>Sally</first_name><last_name>Jones</last_name><email>sally@emailplace.com</email></user><user id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user><user></user></users>`
+
+Query:\
+`_sourceCategory=stag/xmltest ("678" or "emptytag")\
+| limit 5\
+| parse xml "/users/user[3]" as first_name nodrop`
+
+**Old lib **would return <user></user>\
+**New lib** would return <user />\
+2\. Extra spaces in XML tags will be trimmed.
+
+For example - Consider the following xml log line (see the extra space in yellow)\
+`<users><user id="678" role="manager"><first_name>Sally</first_name><last_name>Jones</last_name><email>sally@emailplace.com</email></user><user          id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user></users>`
+
+Query:\
+`<code>_sourceCategory=stag/xmltest ("678" or "emptytag")\
+| limit 5\
+| parse xml "/users/user[2]" as first_name nodrop`
+
+**Old lib** would return: `<user          id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user>`\
+**New lib** would return: `<user id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user><code>`
+
+---
+## October 23, 2020 (Observability)
+
+Update - We have rolled out audit logging support for [Monitors](https://help.sumologic.com/Visualizations-and-Alerts/Alerts/Monitors "Monitors"). Create, read, update, and delete operations of Monitors are logged in the [Audit Event Index](https://help.sumologic.com/Release-Notes/Service-Release-Notes "Service Release Notes").
+
+---
+## October 23, 2020 (Apps)
+
+New - We're happy to announce the [Windows JSON](https://help.sumologic.com/07Sumo-Logic-Apps/04Microsoft-and-Azure/Windows_JSON "https://help.sumologic.com/07Sumo-Logic-Apps/04Microsoft-and-Azure/Windows_JSON") App based on the JSON event log format and provides insight into the operations of the Windows system operation and events so that you can better manage and maintain your environment. The Windows JSON App is based on the JSON Windows event log format and consists of predefined searches and dashboards that provide visibility into your environment for real-time analysis of overall usage of Security Status, System Activity, Updates, User Activity, and Applications.
```

Comment:
```suggestion
New - We're happy to announce the [Windows JSON](https://help.sumologic.com/07Sumo-Logic-Apps/04Microsoft-and-Azure/Windows_JSON) App based on the JSON event log format and provides insight into the operations of the Windows system operation and events so that you can better manage and maintain your environment. The Windows JSON App is based on the JSON Windows event log format and consists of predefined searches and dashboards that provide visibility into your environment for real-time analysis of overall usage of Security Status, System Activity, Updates, User Activity, and Applications.
```